### PR TITLE
Updating VSI tests to use new test framework

### PR DESCRIPTION
**Pull request by jmarolf**

### CODE REVIEW

**Comment by jmarolf** [comment]
test vsi please

**Comment by jmarolf** [comment]
test CI please

File: netci.groovy [hunk 1]
```diff
@@ -146,7 +127,7 @@ set TEMP=%WORKSPACE%\\roslyn-internal\\Open\\Binaries\\Temp
 mkdir %TEMP%
 set TMP=%TEMP%
 
-BuildAndTest.cmd -build:true -clean:false -deployExtensions:true -trackFileAccess:false -officialBuild:false -realSignBuild:false -parallel:true -release:true -delaySignBuild:true -samples:false -unit:false -eta:false -vs:true -cibuild:true -x64:false -netcoretestrun
+Open\\build\\scripts\\cibuild.cmd /release /testVsi
```
Comment (mavasani) [inline netci.groovy]: How does this ensure that we are only running dotnet core VSI tests?

**Comment by davkean** [comment]
Can we turn these on yet?

**Comment by jmarolf** [comment]
I'm testing now on the actual queues.  There are some tests that I may disable while I fix things.  hopefully very soon

**Comment by davkean** [comment]
Before we turn it on PRs, let's do commit jobs and what them for a week.

**Comment by jmarolf** [comment]
Sounds like a plan.  Lets make sure they are stable before turning them on for everyone all the time

**Pull request merged**